atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -159,14 +159,18 @@ def __init__(self, tasker, workflow,
             else:
                 self.log.info("No koji task")
 
+        self.original_koji_task_id = None
         try:
             metadata = get_build_json().get("metadata", {})
-            self.original_koji_task_id = int(metadata['labels']['original-koji-task-id'])
```

Comment:
This line is wrapped into:

```python
try:
    ...
except Exception:
```

Looks the error `"KeyError: 'original-koji-task-id'"` mentioned in issue should have no chance to be raised outside. How can it happen?

I'm on master branch at commit 8f2fc140552835fd7e6a216adb71dc5548fadad3.

Reply:
@tkdchen I'd like to present [this commit](https://github.com/containerbuildsystem/atomic-reactor/commit/fef5b7ed42771f1d199b8fdf2ce2473c0e8689a3#diff-a078811c45cb9386db6ea253e9c97d6d6a3265bf167a5234032fb30413506217), where was the same problem.
Actually, you can reproduce it using the command from the issue description. It looks weird, I know, but it is.

Reply:
exception is logged because of this:

self.log.exception("Failed to fetch original koji task ID")

just change it to self.log.warning 

and no other changes are necessary

all exceptions are catched correctly, be with this we are also logging them

Reply:
Nevertheless, this change describes better what happened, so I'd leave this unchanged